content/issues/2024-09-16-fedora-outage.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Fedora infrastructure outage"
```

Comment:
```suggestion
title: "Failing Copr build jobs"
```

maybe I would explicitly mention what is affected in title

Reply:
sorry, I didn't see your suggestion. If you don't mind I will leave it as it is now...

Reply:
sure, no problem